File: integrations/java/app/revanced/integrations/youtube/patches/EnableTabletLayoutPatch.java
package app.revanced.integrations.youtube.patches;

import app.revanced.integrations.youtube.settings.Settings;

@SuppressWarnings("unused")
public final class EnableTabletLayoutPatch {
    public static boolean enableTabletLayout() {
        return Settings.TABLET_LAYOUT.get();
    }
}

File: patches/bin/main/app/revanced/patches/youtube/layout/tablet/EnableTabletLayoutPatch.kt
package app.revanced.patches.youtube.layout.tablet

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.layout.formfactor.changeFormFactorPatch

@Deprecated("Use 'Change form factor' instead.")
val enableTabletLayoutPatch = bytecodePatch {
    dependsOn(changeFormFactorPatch)
}